midenc-compile/src/stages/link.rs

```diff
@@ -131,8 +155,16 @@ impl Stage for LinkStage {
             #[cfg(not(feature = "std"))]
             InputType::Real(_path) => unimplemented!(),
             InputType::Stdin { name, input } => {
+                // Populate trim_path_prefixes with the current directory
+                let trim_path_prefixes = {
```

Comment:
I think this should be added as a compiler option, just like `current_dir`, so that `cargo-miden` can specify the full set of prefixes it knows about when invoking the compiler (including the location of the Cargo registry for dependency sources).